src/index.js

```diff
@@ -25,7 +25,7 @@ window.onload = () => {
   }
   taskentry.display();
 };
-
+// Clear all completed
 const clearalltask = document.querySelector('.clear-all-tasks');
 clearalltask.addEventListener('click', (e) => {
   e.preventDefault();
```

Comment:
- [ ] When I click on the `Clear all completed` button, It deletes all tasks from the list but It should only delete the tasks marked as completed(tasks with ticked checkboxes). Please make sure to fix this issue.